packages/block-editor/src/components/use-simulated-media-query/index.js

```diff
@@ -22,9 +23,15 @@ function getStyleSheetsThatMatchHostname() {
 	return filter(
 		get( window, [ 'document', 'styleSheets' ], [] ),
 		( styleSheet ) => {
+			if ( ! styleSheet.href ) {
+				return false;
+			}
+			const matcheableUrl = styleSheet.href.match( URL_REGEX );
 			return (
-				styleSheet.href &&
-				styleSheet.href.includes( window.location.hostname )
+				matcheableUrl &&
+				matcheableUrl.length > 2 &&
+				matcheableUrl[ 1 ] === window.location.protocol &&
+				matcheableUrl[ 2 ] === window.location.host
```

Comment:
The url package also has `getAuthority` function, which seems like it returns the same url part as the window.location.host (`subdomain.domain:port`).